src/_P004_Dallas.ino

```diff
@@ -231,6 +225,7 @@ void Plugin_004_DS_startConversion(uint8_t ROM[8])
     for (byte i = 0; i < 8; i++)
         Plugin_004_DS_write(ROM[i]);
     Plugin_004_DS_write(0x44); // Take temperature mesurement
+    delay(800); //give it time to do intial conversion
```

Comment:
No way I am going to merge code which will do 800 msec delay in the reading step of a plugin.

Can you please tell me where this long delay is defined in the datasheet?
I can then make this a multi-step read in which the plugin will try to re-schedule itself when such a timeout has passed.

This delay will block everything and probably cause lots and lots of other issues too.

Reply:
I anticipated this response. The datasheet says 750ms worst case, 800ms is to make sure. With the current scheduler there is no such thing as pre-read (multi-step read). Such a scheduling has to be done on the bus not on a per task basis (even more tricky).

Reply:
We already do something like that.
For example the BME280 also needs some time to  stabilize.
See: 
https://github.com/letscontrolit/ESPEasy/blob/6256cb5255479e564c7fb6ebf1df2bf92c1e43bb/src/_P028_BME280.ino#L286-L295